torch/jit/_trace.py

```diff
@@ -659,14 +666,17 @@ def trace(
             tensors. When a module is passed `torch.jit.trace`, only the
             ``forward`` method is run and traced (see :func:`torch.jit.trace
             <torch.jit.trace_module>` for details).
-        example_inputs (tuple or torch.Tensor):  A tuple of example inputs that
-            will be passed to the function while tracing. The resulting trace
-            can be run with inputs of different types and shapes assuming the
-            traced operations support those types and shapes. `example_inputs`
-            may also be a single Tensor in which case it is automatically
-            wrapped in a tuple.
 
     Keyword arguments:
+        example_inputs (tuple or torch.Tensor or None, optional): A tuple of example
```

Comment:
(related to the other comment above) Can you move these changes to the documentation into `def trace_module` instead of `def trace`?

(Or, if this is already supporting `torch.jit.trace(function, example_kwarg_inputs=...)`, then it would be good anyway to have the `example_kwarg_inputs` option documented in both trace and trace_module).

Reply:
As commented above.